test/automerge-unit.jl

```diff
@@ -616,4 +630,25 @@ end
         result = has_osi_license_in_depot("VisualStringDistances")
         @test !result[1]
     end
+
+    @testset "AutoMerge.linecounts_meet_thresholds" begin
```

Comment:
I think we need to ensure this test only runs on 1.6+ with an `if VERSION >= v"1.6"` check

Reply:
Done.